cockatrice/src/userinfobox.cpp

```diff
@@ -79,6 +86,16 @@ void UserInfoBox::updateInfo(const ServerInfo_User &user)
     else
         userLevelText = tr("Unregistered user");
     userLevelLabel3->setText(userLevelText);
+
+    QString accountAgeString = tr("Unregistered user");
+    if (userLevel.testFlag(ServerInfo_User::IsAdmin) || userLevel.testFlag(ServerInfo_User::IsModerator) || userLevel.testFlag(ServerInfo_User::IsRegistered)) {
+        QDateTime regDate = QDateTime::fromString(QString::fromStdString(user.registrationdate()), "yyyy-MM-dd hh:mm:ss");
+        qint64 accountAgeInDays = regDate.daysTo(QDateTime::currentDateTime());
```

Comment:
Could there be a time zone discrepancy here? The server probably sends back UTC time, does currentDateTime give you UTC or local?